hs_core/views/__init__.py

```diff
@@ -1150,41 +1150,50 @@ def get_metadata_terms_page(request, *args, **kwargs):
 
 
 @login_required
-def get_user_data(request, user_id, *args, **kwargs):
+def get_user_data(request, user_id, is_group, *args, **kwargs):
```

Comment:
I would rename ```user_id``` to ```user_or_group_id``` if we have to have one view function get data for either a user or group. The view function name should be changed to ```get_user_or_group_data```.